File: OOP/Person.java
package in.co.OOP;

public class Person {

	private String name;
	private String address;
	private String city;

	public Person() {
		System.out.println("i'm default Constructor");
	}

	public Person(String name, String address, String city) {

		this.name = name;
		this.address = address;
		this.city = city;
	}

	public String getName() {
		return name;
	}

	public String getAddress() {
		return address;
	}

	public String getCity() {
		return city;
	}

	public static void main(String[] args) {

		Person p = new Person("Raman", "Indore", "Indore");
		System.out.println(p.getName() + p.getCity() + p.getAddress());

	}

}

File: corePython/ConceptsOfOOP/Person.py
class person:

    def __init__(self, name, address, city):
        self.name = name
        self.address = address
        self.city = city

    def getName(self):
        return self.name

    def getAddress(self):
        return self.address

    def getCity(self):
        return self.city

    # def __del__(self):
    #     print("deleting")

    def __str__(self):
        return "%s"%(self.name)

name = input("Enter name:")
address = input("Enter address:")
city = input("Enter city:")


p = person(name,address,city)
print(id(p))
print(p.getName(), p.getAddress(), p.getCity(),p.__str__())

